user/languages/objective-c.md

```diff
@@ -81,9 +87,32 @@ CocoaPods) are added explicitly to the Scheme. To do so:
 
 You will now have a new file in the **xcshareddata/xcschemes** directory
 underneath your Xcode project. This is the shared Scheme that you just
-configured. Check this file into your repository and xctool will be able to
+configured. Check this file into your repository and xcodebuild will be able to
 find and execute your tests.
 
+### Device Destinations
+
+To be able to run tests, xcodebuild needs to know which device to run them on, whether
+that's a real device (the Mac running xcodebuild or an iOS device connected to the Mac)
+or a simulator. When you run tests in Travis CI, you need to tell xcodebuild which
+simulator you want to use by specifying a **device destination**.
+
+Device destinations are strings that identify a particular device to use. You can pass
+them to xcodebuild by using the `-destination` flag. If you're using the default script
+in your Travis CI build, you can use the `xcode_destination` key in your .travis.yml:
+
+```
+xcode_destination: platform=iOS Simulator,OS=11.3,name=iPhone X
+```
+{: data-file=".travis.yml"}
+
+You can find more details about device destinations in the xcodebuild man page.
+
+It's important that your device destination uniquely identifies your device among those
+that Xcode knows about. Since Travis CI's build images have many different simulator
+OS versions installed, you should specify the OS version in your device destination, as
+the name alone is not likely to uniquely identify a single simulator.
```

Comment:
Shall we add some more examples below? Or is this well-known syntax for folks? I sometimes have found that beginners using the UI-only need a bit more help identifying how they can translate that into a command :)

Reply:
It may be well-known for users who already test on the command-line with xcodebuild, but for others it's probably rather foreign. I was attempting to address this by pointing people to the man page for xcodebuild in the previous paragraph, as it includes a lot more information about the syntax and allowed values for destinations.

It's possible that's more indirect than we want to be, though! I would be happy to add a few more example destinations to help users get started more easily when they've only had experience working in the Xcode IDE.